lib/bolt/util/windows/shellwords.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+# Based on the Shellwords Windows fork by Lars Kanis - Licensed under MIT
+# https://github.com/larskanis/shellwords/blob/master/lib/shellwords/msvcrt.rb
```

Comment:
Does this 5 year old shellwords variant represent state of the art in this space? A quick search seems to indicate yes 🐼 

Related Ruby bug - https://bugs.ruby-lang.org/issues/8028

Reply:
Yup 😢

Thus I raised - https://github.com/puppetlabs/bolt/issues/1401